lib/trino-orc/src/test/java/io/trino/orc/TestOrcWriter.java

```diff
@@ -78,6 +89,38 @@ public void testWriteHugeChunk()
         testWriteOutput(columnNameBuilder.build(), data);
     }
 
+    @Test
+    public void testCalendarEntryInFooter()
```

Comment:
Is it feasible to provide a product test to verify that Hive reads the values as expected ?